api/src/pdc/providers/migration/DenoMigrator.ts

```
@@ -0,0 +1,410 @@
+        ${territory_group.company_id}::int
+      )
+      ON CONFLICT DO NOTHING
+      RETURNING *
+    `);
+
+    this.syncSelector(results[0]._id, territory_group.selector);
+  }
+
+  async syncSelector(groupId: number, selector: TerritorySelectorsInterface): Promise<void> {
+    const values: [number[], string[], string[]] = Object.keys(selector)
+      .filter((type) => selector && selector[type as keyof TerritorySelectorsInterface])
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)